server/acl.go

```diff
@@ -35,6 +35,10 @@ func aclMiddleware(s *Server, ce *casbin.Enforcer, next echo.HandlerFunc, readon
 		if common.HasPrefixes(c.Path(), "/api/subscription") && method == "GET" {
 			return next(c)
 		}
+		// Skip GET /api/sql/advise request
```

Comment:
we can declare an open api array so the skip will be more explicit that we just bypass acl for open api

Reply:
Changed the OpenAPI prefix to `/v1` now, so we can skip the check if the request has `/v1` prefix